frontend/stubs/functions.py

```diff
@@ -171,6 +172,11 @@ def pow(x, y):
     return x ** y
 
 
+def range(x: int) -> List[int]:
```

Comment:
I think there should be a second, optional parameter; or can't we have optional parameters in stubs?